api.bs

```diff
@@ -1168,25 +1168,26 @@ to protect against those attacks.
 ### Privacy Budget Epochs ### {#dp-refresh}
 
 Sites receive a separate differential privacy budget for the data in every
-time internval called and epoch. The epoch length is one week.
+time internval called and epoch. The <dfn>privacy budget epoch</dfn> length is
```

Comment:
```suggestion
time interval called an epoch. The <dfn>privacy budget epoch</dfn> length is
```

Reply:
Feel free to ignore the typo fixed by your other PR

Reply:
CC #73